stable/redis/Chart.yaml

```
@@ -1,5 +1,5 @@
 name: redis
-version: 6.0.0
+version: 6.0.1
```

```suggestion
version: 6.0.2
```

The testing bot is failing because of the wrong chart version. I see `6.0.1` is already in master so update this PR to use `6.0.2`